extensions/amp-video-iframe/amp-video-iframe.md

```diff
@@ -191,6 +191,8 @@ The `src` and `poster` URLs are appended with [`data-param-*` attributes as quer
 
 The `/amp-video-iframe` document bootstraps the [integration script](#integration) so that the AMP document can coordinate with the player.
 
+Note: If you're hosting an integration document for users of your video service, feel free to [contribute a code sample to this page.](https://github.com/ampproject/amphtml/blob/master/extensions/amp-video-iframe/amp-video-iframe.md)
+
 Note: For most video providers, `amp-video-iframe` provides enough tools for common playback actions (see [methods](#method) and [events](#postEvent)). Refer to the [vendor player spec](../../spec/amp-3p-video.md) for more details on whether you can use `amp-video-iframe` or you should build a third-party player component instead.
```

Comment:
We probably shouldn't have two notes on top of each other. Lol. I would suggest ordering this section by what you want readers to know first. I think that we want them to host an integration of their service. Maybe we should start the `Third-party video vendors` section with something like:

`AMP accepts vendor integrations in the form of configuration.....`

I'm struggling with the proper wording. But if you get my drift, WDYT?

Reply:
Restructured, let me know if this flows better.